crates/protocol/Cargo.toml

```diff
@@ -8,6 +8,8 @@ license-file.workspace = true
 
 [dependencies]
 chrono = { version = "0.4.31", features = ["serde"] }
+credentials = { git = "https://github.com/TBD54566975/web5-rs", rev = "694db64b72a92e17880c87d31a4bf11fe3bb8793"}
```

Comment:
Will update this once we have packages published. But for now, Cargo makes it really convenient to use a particular repo/commit for a crate dependency!

Reply:
Gotta love that